internal/controller/replication.storage/volumegroupreplication_controller.go

```diff
@@ -111,6 +111,12 @@ func (r *VolumeGroupReplicationReconciler) Reconcile(ctx context.Context, req ct
 		return reconcile.Result{}, err
 	}
 
+	// Skip reconcile if external VGR
+	if instance.Spec.External {
+		r.log.Info("skipping reconcile for non csi-addons managed VGR")
```

Comment:
could be improved as below WDYT?

```suggestion
		r.log.Infof("reconciliation skipped: VGR %s is externally managed", instance.Name)
```

Reply:
If it's just for name then, we are loggin the name just before this check.

Reply:
Ahh, missed the context above 👍